runtime/browser/linux/xwalk_notification_manager.cc

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/browser/linux/xwalk_notification_manager.h"
+
+#include <map>
+
+#include "base/strings/utf_string_conversions.h"
+#include "content/public/browser/browser_thread.h"
+#include "content/public/browser/desktop_notification_delegate.h"
+#include "content/public/browser/web_contents.h"
+#include "content/public/common/platform_notification_data.h"
+#include "third_party/skia/include/core/SkBitmap.h"
+#include "url/gurl.h"
+
+namespace {
+
+using content::BrowserThread;
+using xwalk::XWalkNotificationManager;
+
+// definded by 'org.freedesktop.Notifications'
```

Comment:
Typo: "definded". Plus it should be properly capitalized ("Defined") and end with a period ('.').